vespaclient-container-plugin/src/main/java/com/yahoo/vespa/http/server/FeedHandlerV3.java

```diff
@@ -44,31 +43,40 @@
     private final SessionCache sessionCache;
     protected final ReplyHandler feedReplyHandler;
     private final Metric metric;
+    private final ClientFeederV3.HttpThrottlePolicy httpThrottlePolicy;
     private final Object monitor = new Object();
-    private final AtomicInteger threadsAvailableForFeeding;
     private static final Logger log = Logger.getLogger(FeedHandlerV3.class.getName());
 
-    public FeedHandlerV3(Executor executor,
+    public FeedHandlerV3(ContainerThreadPool threadpool,
                          Metric metric,
                          AccessLog accessLog,
                          DocumentmanagerConfig documentManagerConfig,
                          SessionCache sessionCache,
-                         ThreadpoolConfig threadpoolConfig,
                          DocumentApiMetrics metricsHelper) {
+        this(threadpool.executor(),
+                () -> threadpool.queuedTasks() > 0,
```

Comment:
I believe this would oftentimes be true, even though there's no need to throttle. Consider a larger number than 0.

Reply:
@baldersheim Any input?

Reply:
Hard to tell what is the correct value here. Perhaps queuesize/2 is a more conservative number. Might be better to start with a larger number and reduce. I guess we can  look at queue size afterwards and see how it behaves.

Reply:
I'm not really a fan of this ad-hoc throttling. I think is better to reuse the rejection logic in the threaded request handler base class. The response code must be overridden to 429, but that should be easy to implement.